src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerIndividualValidator.java

```diff
@@ -77,6 +72,40 @@ public Errors validate(List<BeneficialOwnerIndividualDto> beneficialOwnerIndivid
         return errors;
     }
 
+    private void validateAllNatureOfControlFields(BeneficialOwnerIndividualDto beneficialOwnerIndividualDto, Errors errors, String loggingContext) {
+        List<NatureOfControlType> fields = new ArrayList<>();
+        List<NatureOfControlJurisdictionType> jurisdictionFields = new ArrayList<>();
+        if (Objects.nonNull(beneficialOwnerIndividualDto.getBeneficialOwnerNatureOfControlTypes())) {
+            fields.addAll(beneficialOwnerIndividualDto.getBeneficialOwnerNatureOfControlTypes());
+        }
+
+        if (Objects.nonNull(beneficialOwnerIndividualDto.getTrusteesNatureOfControlTypes())) {
+            fields.addAll(beneficialOwnerIndividualDto.getTrusteesNatureOfControlTypes());
+        }
+
+        if (isPropertyAndLandNocEnabled) {
+            if (Objects.nonNull(beneficialOwnerIndividualDto.getNonLegalFirmControlNatureOfControlTypes())) {
+                fields.addAll(beneficialOwnerIndividualDto.getNonLegalFirmControlNatureOfControlTypes());
+            }
+            if (Objects.nonNull(beneficialOwnerIndividualDto.getTrustNatureOfControlTypes())) {
+                fields.addAll(beneficialOwnerIndividualDto.getTrustNatureOfControlTypes());
+            }
+            if (Objects.nonNull(beneficialOwnerIndividualDto.getOwnerOfLandPersonNatureOfControlJurisdictions())) {
+                jurisdictionFields.addAll(beneficialOwnerIndividualDto.getOwnerOfLandPersonNatureOfControlJurisdictions());
+            }
+            if (Objects.nonNull(beneficialOwnerIndividualDto.getOwnerOfLandOtherEntityNatureOfControlJurisdictions())) {
+                jurisdictionFields.addAll(beneficialOwnerIndividualDto.getOwnerOfLandOtherEntityNatureOfControlJurisdictions());
+            }
+        } else {
+            if (Objects.nonNull(beneficialOwnerIndividualDto.getNonLegalFirmMembersNatureOfControlTypes())) {
```

Comment:
I still think this should come out of the `else` block because of the difference with the web implementation. I think it might effect how the Update journey behaves if the flag is switched on. But... guess that if the flag _is_ switched on it's acceptable that Update journey is broken as this (NOCs) is all a work in progress. So will leave it up to you @mattch1 what you do here.